src/main/java/frc/team2412/robot/subsystem/IndexSubsystem.java

```diff
@@ -196,12 +203,14 @@ public boolean isFeederMotorOn() {
     }
 
     /**
-     * Checks if ingest has the correct cargo
+     * Checks if ingest has the correct cargo (also includes the top sensor)
      */
     @Log(name = "Ingest Cargo")
     public boolean ingestHasCorrectCargo() {
         return ((teamColor == Alliance.Blue && ingestBlueColor.get())
-                || teamColor == Alliance.Red && ingestRedColor.get());
+                || teamColor == Alliance.Red && ingestRedColor.get()
```

Comment:
Simpler to read and understand as
(teamColor == Alliance.Blue && ( ingestBlueColor.get() || topBlueColor.get()) ....

Reply:
I see. Thanks